### PR TITLE
OSDOCS-7650: release note MicroShift network changes

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OSDOCS-7650

Link to docs preview:
https://64205--docspreview.netlify.app/microshift/latest/microshift_release_notes/microshift-4-14-release-notes#microshift-4-14-notable-technical-changes

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@pliurh Can you PTAL at the release note about the networking changes? Let me now if lgtm or needs editing? TY!

File: microshift_release_notes/microshift-4-14-release-notes.adoc [hunk 1]
```diff
@@ -125,7 +125,20 @@ This routing rule matches the Kubernetes service IP address range and forwards t
...
+[id="microshift-4-14-traffic-flow-change"]
+==== North-south traffic flow changed
+
+The external gateway bridge and the physical device on the host are no longer connected. The north-south traffic between the network service and the OVN external switch flows from the host kernel to {product-title} through the external gateway bridge. See the xref:../microshift_networking/microshift-networking.adoc#microshift-description-connections-network-topology_microshift-networking[MicroShift networking] documentation.
+
+[discrete]
+[id="microshift-4-14-network-config-flags-deprecated"]
+==== Network configuration flags are deprecated
+
+The host name flag 'gatewayInterface' and the OVS flag `disableOVSInit` in the networking configuration file, `/etc/microshift/ovn.yaml/etc/microshift/ovn.yaml`, are deprecated with this release.
```
Comment (pliurh) [inline microshift_release_notes/microshift-4-14-release-notes.adoc:140]: It is not a host name flag.

Comment (ShaunaDiaz) [reply microshift_release_notes/microshift-4-14-release-notes.adoc:140]: @pliurh What it is, then, please?

**Comment by ShaunaDiaz** [comment]
@pliurh Is this lgtm?

**Comment by ShaunaDiaz** [comment]
@dhensel-rh Is QE good with this release note?

File: microshift_release_notes/microshift-4-14-release-notes.adoc [hunk 2]
```diff
@@ -125,7 +125,19 @@ This routing rule matches the Kubernetes service IP address range and forwards t
...
 === Networking changes
 
-Networking changes to {product-title} {product-version} include IP address assignment changes, gateway changes, and other refinements.
+Networking updates to {product-title} {product-version} include traffic flow pattern, gateway, and custom configuration changes.
+
+[discrete]
+[id="microshift-4-14-traffic-flow-change"]
+==== North-south traffic flow changed
+
+The external gateway bridge and the physical device on the host are no longer connected. The north-south traffic between the network service and the OVN external switch flows from the host kernel to {product-title} through the external gateway bridge. See the xref:../microshift_networking/microshift-networking.adoc#microshift-description-connections-network-topology_microshift-networking[MicroShift networking] documentation.
```
Comment (skopacz1) [inline microshift_release_notes/microshift-4-14-release-notes.adoc:134]: Suggestion:

```suggestion
The external gateway bridge and the physical device on the host are no longer connected. The north-south traffic between the network service and the OVN external switch flows from the host kernel to {product-title} through the external gateway bridge. See the xref:../microshift_networking/microshift-networking.adoc#microshift-description-connections-network-topology_microshift-networking[MicroShift networking] documentation for more information.
```

File: microshift_release_notes/microshift-4-14-release-notes.adoc [hunk 3]
```diff
@@ -125,7 +125,19 @@ This routing rule matches the Kubernetes service IP address range and forwards t
...
+
+[discrete]
+[id="microshift-4-14-network-config-flags-deprecated"]
+==== Network configuration flags are deprecated
+
+The gateway bridge flag, 'gatewayInterface', and the OVS flag, `disableOVSInit`, in the networking configuration file, `/etc/microshift/ovn.yaml/etc/microshift/ovn.yaml`, are deprecated with this release. See the xref:../microshift_networking/microshift-networking.adoc#microshift-config-OVN-K_microshift-networking[MicroShift OVN-K configuration] documentation.
```
Comment (skopacz1) [inline microshift_release_notes/microshift-4-14-release-notes.adoc:140]: Same suggestion as above:

```suggestion
The gateway bridge flag, 'gatewayInterface', and the OVS flag, `disableOVSInit`, in the networking configuration file, `/etc/microshift/ovn.yaml/etc/microshift/ovn.yaml`, are deprecated with this release. See the xref:../microshift_networking/microshift-networking.adoc#microshift-config-OVN-K_microshift-networking[MicroShift OVN-K configuration] documentation for more information.
```

Comment (skopacz1) [inline microshift_release_notes/microshift-4-14-release-notes.adoc:140]: I think you duplicated the file name here:

```suggestion
The gateway bridge flag, 'gatewayInterface', and the OVS flag, `disableOVSInit`, in the networking configuration file, `/etc/microshift/ovn.yaml`, are deprecated with this release. See the xref:../microshift_networking/microshift-networking.adoc#microshift-config-OVN-K_microshift-networking[MicroShift OVN-K configuration] documentation.
```

**Review by skopacz1 — COMMENTED** [review]
Left a few comments, otherwise looks good to me!

**Comment by ShaunaDiaz** [comment]
Note other MicroShift relnote PRs are also in the merge review queue.

**Review by jeana-redhat — APPROVED** [review]
/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Pull request merged**